index.html

```
@@ -676,6 +676,8 @@ <h3 class="font-headline font-extrabold text-lg sm:text-xl">2026 Timeline</h3>
 <section id="orgs" class="px-6 max-w-7xl mx-auto pb-20">
   <div class="flex items-center justify-between mb-8">
     <p class="font-label text-xs uppercase tracking-widest text-zinc-500">Showing <strong id="orgCount">184</strong> of 184 organizations</p>
+    <div class="flex flex-wrap items-center gap-4">
+      <select id="categoryFilter" class="bg-surface-container-low border-none rounded-xl text-xs font-bold focus:ring-0 cursor-pointer text-zinc-600">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </select> ], start tag match failed [ <select id="categoryFilter" class="bg-surface-container-low border-none rounded-xl text-xs font-bold focus:ring-0 cursor-pointer text-zinc-600"> ] on line 680.

(tag-pair)